src/components/Cart/PriceCard.js

```diff
@@ -0,0 +1,45 @@
+import { useCart } from "../../context/cart-context";
+
+export const PriceCard = () => {
+    const {cartState} = useCart();
+
+    let totalPricetoPay = cartState.reduce((sum, curr)=>sum + curr.quantity * curr.price, 0)
+
```

Comment:
make constant

Reply:
const can be used if reassigning of variable is not happening